test/validations.test.js

```diff
@@ -322,6 +322,43 @@ describe('validations', function() {
       });
     });
 
+    it('should pass options to custom synch', function(done) {
+      delete User.validations;
+      User.validate('name', function(err, options) {
+        if (options.options !== 'someValue') err();
+      });
+      User.create({name: 'Valid'}, {options: 'someValue'}, function(e, d) {
+        d.updateAttribute('name', null, {options: 'otherValue'}, function(e) {
+          should.exist(e);
+          e.should.be.instanceOf(Error);
+          e.should.be.instanceOf(ValidationError);
+          d.updateAttribute('name', 'Vasiliy', {options: 'someValue'}, function(e) {
+            should.not.exist(e);
+            done();
+          });
+        });
+      });
+    });
+
+    it('should pass options to custom async', function(done) {
+      delete User.validations;
+      User.validateAsync('name', function(err, options, done) {
+        if (options.options !== 'someValue') err();
```

Comment:
Same comment as above applies here too.

Additionally, your code will call both `err` and `done`, which does not look correct to me. I think you should `return` after calling `err()`.

Reply:
nope, thats correct, you call err() then done()....